action_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func init() {
-	os.Unsetenv("LS_COLORS")
+	_ = os.Unsetenv("LS_COLORS")
 }
```

Comment:
While I understand why you do this, I have to raise the point it's not normal to find an init this in the code.

```diff
-func init() { 
-        os.Unsetenv("LS_COLORS") 
+func TestMain(m *testing.M) {
+       _ = os.Unsetenv("LS_COLORS") 
+       os.Exit(m.Run())
  }
```

Reply:
I know your PR was focused on fixing linting issues only, but you should consider fixing what I reported here.

TestMain should be used to alter the behavior of test env.

Here the init func should be replaced

Reply:
ah yes, missed that